pyccl/cls.py

```diff
@@ -292,6 +293,8 @@ def angular_cl(cosmo, cltracer1, cltracer2, ell,
         cltracer1, cltracer2 (:obj:`Tracer`): Tracer objects, of any kind.
         ell (float or array_like): Angular wavenumber(s) at which to evaluate
             the angular power spectrum.
+        p_of_k_a (Pk2D object or None): 3D Power spectrum to project. If None,
```

Comment:
We should use the correct sphinx syntax to link things here.

Reply:
Fixed